src/otx/algo/classification/efficientnet_b0.py

```diff
@@ -10,8 +11,27 @@
     MMPretrainMultilabelClsModel,
 )
 
+class EfficientNetB0LoadWeightMixin:
+    def convert_previous_otx_ckpt_impl(
+        self, 
+        state_dict: dict[str, Any],
+        label_type: str,
+        add_prefix: str = "", 
+    ):
+        """Convert the previous OTX ckpt according to OTX2.0."""
+        for key in list(state_dict.keys()):
+            val = state_dict.pop(key)
+            if key.startswith("features.") and "activ" not in key:
+                key = "backbone." + key
+            elif key.startswith("output."):
+                key = key.replace("output", "head")
+                if not label_type == "hlabel":
+                    key = key.replace("asl", "fc")
+                val = val.t()
+            state_dict[add_prefix + key] = val
+        return state_dict 
 
-class EfficientNetB0ForHLabelCls(MMPretrainHlabelClsModel):
+class EfficientNetB0ForHLabelCls(EfficientNetB0LoadWeightMixin, MMPretrainHlabelClsModel):
```

Comment:
It seems too much to introduce mixin into this. I still think that the complexity by adding this is not necessary to solve our problem.
```suggestion
class EfficientNetB0ForHLabelCls(MMPretrainHlabelClsModel):
```

Reply:
Changed.